src/main/resources/com/zestedesavoir/zestwriter/fxml/Editor.fxml

```diff
@@ -145,6 +145,9 @@
             <StyleClassedTextArea fx:id="SourceText" wrapText="true" BorderPane.alignment="CENTER" >
             </StyleClassedTextArea>
          </center>
+         <bottom>
+             <Label text="Nombre de mots :" fx:id="wordcount"></Label>  
```

Comment:
Oups.

Le 4 mai 2016 à 11:12, firm1 <notifications@github.com> a écrit :

> In src/main/resources/com/zestedesavoir/zestwriter/fxml/Editor.fxml
> <https://github.com/firm1/zest-writer/pull/98#discussion_r62009672>:
>
> > @@ -145,6 +145,9 @@
> >              <StyleClassedTextArea fx:id="SourceText" wrapText="true" BorderPane.alignment="CENTER" >
> >              </StyleClassedTextArea>
> >           </center>
> > +         <bottom>
> > +             <Label text="Nombre de mots :" fx:id="wordcount"></Label>
>
> il faut remplacer wordcount par WordCountLabel pour que ça fonctionne.
> C'est comme ça que tu l'as appelé dans le controlleur.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/firm1/zest-writer/pull/98/files/07350efcf726081f7b3e4e153dcf807571ff941f#r62009672>
>